codegenerator/cli/npm/envio/index.js

```diff
@@ -0,0 +1,37 @@
+// This file is needed to have control over TS version exports
+// Some parts like Sury reexport are impossible to implement
+// on the JS side, so we need to do it here
+
+const envioGen = require("./src/Envio.bs.js");
+Object.assign(exports, envioGen);
+
+const Sury = require("rescript-schema");
+// Important! Should match the index.d.ts file
+exports.S = {
+  string: Sury.string,
+  jsonString: Sury.jsonString,
+  boolean: Sury.boolean,
+  int32: Sury.int32,
+  number: Sury.number,
+  bigint: Sury.bigint,
+  never: Sury.never,
```

Comment:
We can write a test that iterates over the exports and validates the types

Reply:
There were tests using rescript-schema for fuel, I turned them to use the re-exported module `S`.